### PR TITLE
TATB - 36 : redesign login and registration form

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**

### CODE REVIEW

File: src/app/authorization/login/login.component.css [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+  border: none;
+  border-bottom: 1px solid #d1d1d4 !important;
+}
+
+.card {
+  padding: 10px;
+  margin: 0 auto;
+  height: auto;
+}
+.d-grid {
```
Comment (charmijtdev) [inline src/app/authorization/login/login.component.css:31]: why used boostrap class direct

File: src/app/authorization/login/login.component.css [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+  margin-top: 20px;
+}
+.btn.btn-primary {
```
Comment (charmijtdev) [inline src/app/authorization/login/login.component.css:34]: again here too pelase remove this and use custom class instead or boostrpa class

File: src/app/authorization/registration/registration.component.css [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
-}
-input[type="date"]{
-    margin-left:20px;
-    padding:3px;
-    font-family: "Roboto Mono",monospace;
-    border-radius: 4px;
-    width:320px;
-    margin-top: 10px;
+
+.fs-3 {
```
Comment (charmijtdev) [inline src/app/authorization/registration/registration.component.css:5]: boostrap class please remove this

File: src/app/authorization/registration/registration.component.css [hunk 4]
```diff
@@ -1,33 +1,34 @@
...
+  font-weight: bold;
+  color: rgb(63, 63, 236);
 }
-.cpwd{
-    margin-top: 10px;
+.card {
```
Comment (charmijtdev) [inline src/app/authorization/registration/registration.component.css:9]: boostrap class remove this

File: src/app/authorization/registration/registration.component.css [hunk 5]
```diff
@@ -1,33 +1,34 @@
...
+  padding: 10px;
+  margin: 0 auto;
+  margin-bottom: 0px;
+  height: auto;
 }
-.gender{
-    margin-top: 10px;
+
+.form-control {
```
Comment (charmijtdev) [inline src/app/authorization/registration/registration.component.css:16]: boostrap class remove thsi

File: src/app/authorization/registration/registration.component.css [hunk 6]
```diff
@@ -1,33 +1,34 @@
...
+  width: 100%;
+  border-bottom: 2px solid #d1d1d4 !important;
+  border: none;
 }
-.col-md-3{
-    width:100%;
-    float:left;
-    margin:0;
+
+.btn.btn-success {
```
Comment (charmijtdev) [inline src/app/authorization/registration/registration.component.css:25]: boostrap class remove this

File: src/app/component/footer/footer.component.css [hunk 7]
```diff
@@ -1,27 +1,25 @@
...
+  color: #0000EE;
 }
 
-@media (min-width: 1300px) {
-    .footer {
-        font-size: 14px !important;
-    }
-}
\ No newline at end of file
+.fa {
```
Comment (charmijtdev) [inline src/app/component/footer/footer.component.css:15]: why direct icon class

File: src/app/component/header/header.component.html [hunk 8]
```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html:1]: Please remove this inline css

File: src/app/component/header/header.component.html [hunk 9]
```diff
@@ -1,10 +1,10 @@
...
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
-      <img class="d-inline-block" src="assets/img/logo1.png" width="35" height="35" />
+      <img class="d-inline-block" src="assets/img/logo1.png" alt="Jyoti Techn" width="35" height="35" />
```
Comment (charmijtdev) [inline src/app/component/header/header.component.html:4]: remove line css

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/authorization/login/login.component.css (hunk 1 above)
Comment (PrabhatJTDeveloper) [reply src/app/authorization/login/login.component.css:31]: I have Remove bootstrap class

File: src/app/authorization/login/login.component.css (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/authorization/login/login.component.css:34]: yes I have Remove bootstrap class in CSS and I have uses to custom class

File: src/app/authorization/registration/registration.component.css (hunk 6 above)
Comment (PrabhatJTDeveloper) [reply src/app/authorization/registration/registration.component.css:25]: yes I have Remove Bootstrap class

File: src/app/authorization/registration/registration.component.css (hunk 5 above)
Comment (PrabhatJTDeveloper) [reply src/app/authorization/registration/registration.component.css:16]: I have uses to input filed class name form-control but I have change class name form

File: src/app/authorization/registration/registration.component.css (hunk 4 above)
Comment (PrabhatJTDeveloper) [reply src/app/authorization/registration/registration.component.css:9]: yes I have change class

**Comment by charmijtdev** [comment]
OLD PR

**Pull request closed without merging**